src/Algolia.Search/Models/Analytics/Variant.cs

```diff
@@ -89,5 +89,10 @@ public class Variant
         /// Search parameters for AA Testing
         /// </summary>
         public Query CustomSearchParameters { get; set; }
+
+        /// <summary>
+        /// Tracked search count
+        /// </summary>
+        public int? TrackedSearchCount { get; set; }
```

Comment:
Did you double check if it's a int or a long (int64)?

Reply:
yes :)